_includes/image

```diff
@@ -0,0 +1,12 @@
+<figure class="{{ include.class }}">
+  <img src=
+    {% if include.image_path contains "://" %}
+      "{{ include.image_path }}"
+    {% else %}
+      "{{ include.image_path | prepend: "/assets/images/" | absolute_url }}"
```

Comment:
One more minor thing. With the latest MM I moved away from assuming images would be placed in `/assets/images/` and instead encourage adding the full path. Adds more flexibility to where they can be placed for the user.

If you could remove `prepend: "/assets/images/"` it fall in line with the [other helpers](https://mmistakes.github.io/minimal-mistakes/docs/helpers/) and I can merge it in.